Choclate

MrChoclate opened the pull request.

MrChoclate committed:
Add photos

MrChoclate committed:
List advance features and add django traceback

MrChoclate committed:
Add elemental list operations. Add dict manipulation.

MrChoclate commented:
@Atilla106/core @Pozuelomar review

Thooms commented on public/index.html:
```diff
@@ -136,6 +159,10 @@
 					<h5>Ajoutez un cache à vos fonctions !</h5>
+					<span>
+						Un décorateur est une fonction qui prend en paramètre une fonction
+						et qui renvoie une fonction
```
Peut-être pas assez comme explication, si ?

MrChoclate replied:
J'y ai pensé, effectivement il faut au moins la traduction du sucre syntaxique

Thooms commented on public/index.html:
```diff
@@ -153,9 +180,43 @@
+				<section>
+					<h5>Vous n'avez jamais autant aimé chercher dans un dictionaire ! (2)</h5>
+					<ul>
+						<li>Très utile pour représentez une liste d'adjacence (graphe)</li>
```
représenter*

Thooms commented on public/index.html:
```diff
@@ -167,6 +228,25 @@
+						<li class="fragment">Meta classe</li>
+						<li class="fragment">Binding C</li>
+						<li class="fragment">Fonctionnement interne des structures de données</li>
+					</ul>
```
générateurs ?

MrChoclate replied:
J'avais pour objectif de l'aborder à la fin, je l'ajoute dans le doute

MrChoclate committed:
@Thooms review.

Add decorator details. Fix typos. Add a slide to sum up the data
structures seen so far.
